### ISSUE TITLE
Workshop node

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#56]
_TL;DR: Create a node that helps to schedule workshop classes._

A class is a group of students.

- [ ] The class should have:
  - name (string)
  - id (string)
  - startingAt (date as ISO string)
  - participantIds: string[]
- [ ] Each participant should have the following properties:
  - id (string)
  - fullname (string)
  - email (string)
  - duration (participants from a class can join only for a couple of days instead of the full course). (string)
  - fromCompany (string)
  - confirmed (boolean)
  - paid (boolean)
  - hasAccessToAcademy (boolean)
  - isPrerequisitesShared (boolean)
  - notes (string)
  - isCertificateShared (boolean)

## Messages

You should add messages for:
- adding participants `UPSERT_PARTICIPANT.V1` and classes `UPSERT_CLASS.V1`.
- deleting participants `DELETE_PARTICIPANT.V1` and classes `DELETE_CLASS.V1`.
- You should add a `LIST_CLASSES.V1` message to list all the classes filtered by `before` and `since` times.

## Events

An event to list all classes and participants.
Please note that the participants should be inlined with the class object (no `participantIds`, but a full participant object).
